### PR TITLE
[MIG][13.0] base_external_dbsource_sqlite: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT20715
@carlosdauden @chienandalu Review please!!
This PR denpends on this https://github.com/OCA/server-backend/pull/54

### CODE REVIEW

File: base_external_dbsource_sqlite/__manifest__.py [hunk 1]
```diff
@@ -2,24 +2,14 @@
...
-        ],
-    },
-    'demo': [
-        'demo/base_external_dbsource.xml',
-    ],
-    'installable': True,
+    "name": "External Database Source - SQLite",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "author": "Daniel Reis, " "LasLabs, " "Odoo Community Association (OCA)",
```
Comment (chienandalu) [inline base_external_dbsource_sqlite/__manifest__.py]:
```suggestion
    "author": "Daniel Reis, LasLabs, Odoo Community Association (OCA)",
```

**Review by chienandalu — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@chienandalu Changes done!!

**Comment by chienandalu** [comment]
Now pre-commit is failing on an xml file

**Comment by pedrobaeza** [comment]
@sergio-teruel if you want, we can meet for explaining what to do in these cases where the migration started before all the pre-commit stuff

**Comment by sergio-teruel** [comment]
@pedrobaeza Thanks... I forgot do a rebase....

**Pull request merged**